docs/runtimes/configuring-job-resources.md

```diff
@@ -363,8 +363,11 @@ func.run(state_thresholds={"running": "1 min", "image_pull_backoff": "1 minute a
 ```
 
 See:
+
+- {py:meth}`mlrun.runtimes.RemoteRuntime.set_state_thresholds`
```

Comment:
I don't think this line needs to be referenced. It just blocks it for nuclio runtime and you mention it below. That's enough